tests/Database/DatabaseEloquentFactoryTest.php

```diff
@@ -441,6 +441,22 @@ public function test_resolve_non_app_nested_model_factories()
         }
     }
 
+    public function test_resolve_package_model_factories(): void
+    {
+        Factory::useNamespace('Vendor\\Package\\Database\\Factories\\');
+
+        $resolves = [
+            'Vendor\\Package\\Models\\Foo' => 'Vendor\\Package\\Database\\Factories\\FooFactory',
+            'Vendor\\Package\\Models\\Bar' => 'Vendor\\Package\\Database\\Factories\\BarFactory',
+            'Vendor\\Package\\Models\\Sub\\Foo' => 'Vendor\\Package\\Database\\Factories\\FooFactory',
```

Comment:
If `Vendor\Package\Models\Sub\Foo` is a different model than `Vendor\Package\Models\Foo`, shouldn't they use different factory classes?

Reply:
you are right. My Idea on it wasn't to change the actual implementation to much. Would i implement on my pull request a resolving with sub folders, so anyone would need to write an additional entry inside of the documentation for it (i thought).